test/check-machines-consoles

```diff
@@ -99,6 +99,7 @@ class TestMachinesConsoles(VirtualMachinesCase):
     def testInlineConsoleWithUrlRoot(self, urlroot=""):
         self.testInlineConsole(urlroot="/webcon")
 
+    @skipImage('TODO: setlocale: No such file or directory', "arch")
```

Comment:
How can we fix this without blocking on a new upstream glibc? Could we set `LANG=en_US.UTF-8` for arch?

FTR, I'm totally fine with this TODO skip, I just wouldn't want to keep it around for a year.

Reply:
Should be fixed in February, but yeah still need to figure this one out. Skipped it for now to not block the whole PR.